src/js/components/TextInput/TextInput.js

```diff
@@ -268,6 +269,11 @@ class TextInput extends Component {
 
   onChange = event => {
     const { onChange } = this.props;
+    if (event.target.value.length > 0) {
```

Comment:
What do you think of `this.setState({ isInputEmpty: !Boolean(event.target.value) })` ?

Reply:
Good idea, or maybe just `!!event.target.value`

Reply:
so more elegant!!
tnks :)